jerry-core/jcontext/jcontext.h

```diff
@@ -143,6 +143,10 @@ typedef struct
 } jerry_hash_table_t;
 #endif /* !CONFIG_ECMA_LCACHE_DISABLE */
 
+/**
+ * IF EXTERNAL_CONTEXT is not enable then INIT_CURRENT_INSTANCE is a empty macro to avoid warnings
```

Comment:
- Typos: "If", "enabled"
- Placement: this docstring is completely unrelated to the syntactically following entity (`JERRY_GET_CURRENT_INSTANCE`). If `INIT_CURRENT_INSTANCE` is to be documented then docstring should precede that macro.
- Semantics: I don't think that the macro is empty in some cases to avoid warnings but not to bloat code, isn't it?